lib/portlayer/task/inspect.go

```diff
@@ -35,42 +34,33 @@ func Inspect(op *trace.Operation, h interface{}, id string) (*executor.SessionCo
 	stasks := handle.ExecConfig.Sessions
 	etasks := handle.ExecConfig.Execs
 
-	_, okS := stasks[id]
-	_, okE := etasks[id]
-
 	op.Debugf("target task ID: %s", id)
 	op.Debugf("session tasks during inspect: %s", stasks)
-	op.Debugf("exec tasks during inspect: %s", etasks)
 
-	if !okS && !okE {
-		if handle.Runtime.PowerState == types.VirtualMachinePowerStatePoweredOff {
-			powerStateError := TaskPowerStateError{
-				msg: fmt.Sprintf("the operation cannot be completed, container(%s) has been shut down during the operations execution.", handle.ExecConfig.ID),
-			}
-			return nil, powerStateError
-		}
+	// print all of them, otherwise we will have to assemble the id list regardless of
```

Comment:
Looking at this more, I think I was trying to avoid looking through the lists. But the lists come from the handle so they will all be associated with the correct call(or should be barring some other failure). I am going to remove this comment entirely.